src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
+import java.util.Random;
 
 public class ColorSupplier {
+    Random random = new Random();
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
There's no need to create a new instance of 'Random' here since you already have a 'random' field in this class. Use the existing 'random' instance to call 'nextInt'.